include/inexor/vulkan-renderer/wrapper/device.hpp

```diff
@@ -14,12 +14,12 @@ namespace inexor::vulkan_renderer::wrapper {
 class Device {
     VkDevice m_device;
     VkPhysicalDevice m_graphics_card;
-    VmaAllocator m_allocator;
-    std::string m_gpu_name;
+    VmaAllocator m_allocator{VK_NULL_HANDLE};
+    std::string m_gpu_name{""};
 
-    VkQueue m_graphics_queue;
-    VkQueue m_present_queue;
-    VkQueue m_transfer_queue;
+    VkQueue m_graphics_queue{nullptr};
+    VkQueue m_present_queue{nullptr};
+    VkQueue m_transfer_queue{nullptr};
```

Comment:
As i just see this, how do we handle the default initialization stuff?
 Do we init with `{}` with `{""}` or just nothing, here the string would be `""` anyway.
@IAmNotHanni @yeetari

Reply:
For the string, I wouldn't put anything. `std::string`'s default constructor initializes it to an empty string anyway.

Reply:
I added this to https://github.com/inexorgame/vulkan-renderer/issues/218.